src/main/scala/io/iohk/ethereum/network/p2p/messages/CommonMessages.scala

```diff
@@ -111,10 +118,23 @@ object CommonMessages {
     )
 
     val FirstByteOfAddress = 12
-    val LastByteOfAddress = 32
+    val LastByteOfAddress = FirstByteOfAddress + AddressLength
```

Comment:
Should this be part of the `Transaction`? If at all I would say it goes better in the comp object. In the future I think it would useful to have an `Address` class.